### ISSUE TITLE
ahk is unable to know the value of a field if it is "default"

### ISSUE THREAD

**Issue by lipkau** [ewerybody/a2#131]
### Expected Behavior
AHK is able to know the value of a field/element at all times

### Current Behavior
AHK is able to know the value of a field if it was changed

### Possible Solution
1. leave as is and document this **really** well.  
    * workaround:  
```
isCheckBoxActive := this.manifest.findUIElementByKey("name", "MyModule_CheckBox1")
```
2.  write the values into the ahk runtime even if unchanged from default

**Comment by lipkau** [comment]
my bad.
value is available in variables.
what is not available is `combo_String1` as the DB entry

**Comment by ewerybody** [comment]
Was that fixed with #130?
